toolbox/workflow/guest_start_nvme.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+"""
+Start a qemu-guest with NVMe devices
+====================================
+
```

Comment:
- [ ] Sign-off is missing on this commit
- [ ] Adjust worklet according to below

a) Move into ``xnvme/toolbox/cijoe-pkg-xnvme/src/cijoe/xnvme/worklets``, then others can use it by referring to it by ``uses: xnvme.guest_start_nvme``.
**or**
b) Add the worklet changes on https://github.com/refenv/cijoe-pkg-qemu, such that the other users of it benefits from the refactoring and addition of the "large-mdts" device.